apis/config/v1beta1/validation_test.go

```diff
@@ -72,15 +87,20 @@ func TestPodIntegrationOptions_ValidateNamespaceSelector(t *testing.T) {
 
 	for name, tc := range testCases {
 		t.Run(name, func(t *testing.T) {
-			kueueNamespace := "kueue-system"
-			gotError := tc.podIntegrationOptions.ValidateNamespaceSelector(&kueueNamespace)
+			cfg := Configuration{
+				Namespace: ptr.To("kueue-system"),
+				Integrations: &Integrations{
+					PodOptions: tc.podIntegrationOptions,
+				},
+			}
+			gotError := cfg.validateNamespaceSelector()
 
 			if tc.wantError == nil {
 				if diff := cmp.Diff(tc.wantError, gotError); diff != "" {
 					t.Errorf("unexpected non-nil error (-want,+got):\n%s", diff)
 				}
 			} else {
-				if diff := cmp.Diff(tc.wantError.Error(), gotError.Error()); diff != "" {
+				if diff := cmp.Diff(tc.wantError, gotError, cmpopts.IgnoreFields(field.Error{})); diff != "" {
```

Comment:
```suggestion
				if diff := cmp.Diff(tc.wantError, gotError, cmpopts.IgnoreFields(field.Error{}, "Value", "Message")); diff != "" {
```

Reply:
I would rather just skip `IgnoreFields` option for now, since there's no difference in those fields.